pkgs/by-name/pw/pwndbg/package.nix

```diff
@@ -7,47 +7,43 @@
 }:
 
 let
-  pythonPath = with python3.pkgs; makePythonPath [
-    capstone
-    future
-    psutil
-    pwntools
-    pycparser
-    pyelftools
-    pygments
-    unicorn
-    rpyc
-  ];
+  pythonPath = with python3.pkgs; makePythonPath [ pwndbg ];
+
   binPath = lib.makeBinPath ([
-    python3.pkgs.pwntools   # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/wrappers/checksec.py#L8
+    python3.pkgs.pwntools # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/wrappers/checksec.py#L8
   ] ++ lib.optionals stdenv.isLinux [
-    python3.pkgs.ropper     # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/ropper.py#L30
-    python3.pkgs.ropgadget  # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/rop.py#L32
+    python3.pkgs.ropper # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/ropper.py#L30
+    python3.pkgs.ropgadget # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/rop.py#L32
   ]);
-
-in stdenv.mkDerivation rec {
+in
+stdenv.mkDerivation rec {
   pname = "pwndbg";
-  version = "2022.12.19";
+  version = "2024.02.14";
   format = "other";
 
   src = fetchFromGitHub {
```

Comment:
Can we use same `src` from `pkgs/development/python-modules/pwndbg/default.nix`?

Reply:
Capstone use same syntax- https://github.com/NixOS/nixpkgs/blob/nixos-23.11/pkgs/development/python-modules/capstone/default.nix#L12